### ISSUE TITLE
Does not support cross account resource access

### ISSUE THREAD

**Issue by szeevs** [airflow-plugins/redshift_plugin#6]
I have modified for cross account access to work.
Here is my example:
```
def getS3Conn():
            creds = ""
            s3_conn = get_conn(self.s3_conn_id)
            aws_key = s3_conn.extra_dejson.get('aws_access_key_id')
            aws_secret = s3_conn.extra_dejson.get('aws_secret_access_key')
            # support for cross account resource access
            aws_role_arn = s3_conn.extra_dejson.get('role_arn')
            if aws_key and aws_secret:
                creds = ("aws_access_key_id={0};aws_secret_access_key={1}"
                    .format(aws_key, aws_secret))
            elif aws_role_arn:
                creds = ("aws_iam_role={0}"
                    .format(aws_role_arn))
            return creds
```

**Comment by benjamingregory** [comment]
Thanks @szeevs - I'm not too familiar with cross account resource access.

Can you take what you've adapted and submit a pr?

**Comment by szeevs** [comment]
OK. I will do that. No problem.

**Comment by szeevs** [comment]
Closed with b275fd4
